OpenRA.Mods.TS/UtilityCommands/LegacyTilesetImporter.cs

```diff
@@ -57,7 +59,24 @@ public void Run(ModData modData, string[] args)
 				"Cliff" // TS defines this as "Rock"
 			};
 
+			var metadata = new StringBuilder();
+			metadata.AppendLine("General:");
+
+			var name = Path.GetFileNameWithoutExtension(args[1]);
```

Comment:
Can we please make this a proper commandline argument instead of hacking it from the filename (which then fails 50% of the time in ts!)?